### PR TITLE
Bug 1701203 - Declare a minimum node version in the package.json

### PR DESCRIPTION
**Pull request by AmrDhaliwal**
Fixed the bug that required a minimum node version by adding the engine field to package.json.

### CODE REVIEW

**Comment by Dexterp37** [comment]
Hey @AmrDhaliwal , thank you for your PR and welcome!

This PR is currently failing our CI tests. Would you kindly check why that's happening? You can also run some tests locally with `npm run test:core`.

**Review by AmrDhaliwal — COMMENTED** [review]
Removed an extra comma from line 103 in package.json in glean that was causing errors.

**Comment by AmrDhaliwal** [comment]
Hi @Dexterp37,

It seems there was an extra comma that was causing the problems.
But, I removed it and all the tests are now passing!

File: glean/package.json [hunk 1]
```diff
@@ -96,5 +96,9 @@
   "dependencies": {
     "jose": "^3.7.0",
     "uuid": "^8.3.2"
+  },
+  "engines": {
+    "node": ">=12.20.0",
+    "npm": ">7.0.0"
```
Comment (Dexterp37) [inline glean/package.json:102]: I think what we really want is `>=` here

**Review by Dexterp37 — CHANGES_REQUESTED** [review]
Thanks for making the tests pass! I have one additional small request.

File: glean/package.json [hunk 2]
```diff
@@ -96,5 +96,9 @@
   "dependencies": {
     "jose": "^3.7.0",
     "uuid": "^8.3.2"
+  },
+  "engines": {
+    "node": ">=12.20.0",
```
Comment (Dexterp37) [inline glean/package.json:101]: Would you kindly add an entry to our  `CHANGELOG.md` file?

File: glean/package.json (hunk 1 above)
Comment (AmrDhaliwal) [reply glean/package.json]: I have gone ahead and made that change.

File: glean/package.json (hunk 2 above)
Comment (AmrDhaliwal) [reply glean/package.json:101]: Hi @Dexterp37,

I have added the bug as an entry into the CHANGELOG.md file.

File: glean/package-lock.json [hunk 3]
```diff
@@ -41,6 +41,10 @@
         "web-ext-types": "^3.2.1",
         "webpack": "^5.34.0",
         "webpack-cli": "^4.5.0"
+      },
+      "engines": {
+        "node": ">=12.20.0",
+        "npm": ">7.0.0"
```
Comment (Dexterp37) [inline glean/package-lock.json:47]: Please run `npm i --package-lock-only` to update the lock file

**Review by Dexterp37 — CHANGES_REQUESTED** [review]
We're almost there! Thanks again for this!

**Comment by AmrDhaliwal** [comment]
Hi @Dexterp37,

I updated the CHANGELOG.md and updated the lock file.
Sorry this is taking me so long and thanks for the help.

**Review by Dexterp37 — APPROVED** [review]
Thank you! And don't worry about the delays, there's no rush :)

**Pull request merged**